### DISCUSSION TITLE
请问我要用DB网络检测以下类型含有上下偏差的数据，那种框选方式是可行的？

### DISCUSSION THREAD

**Discussion by Gebbap** [PaddlePaddle/PaddleOCR#14025] [Q&A]
![框选方式一](https://github.com/user-attachments/assets/0dd9991c-2752-43ce-a232-a9d0391a26e1)
![框选方式二](https://github.com/user-attachments/assets/c77980a5-116e-42d0-95b2-3f1c8bd1b6f5)

**Comment by Liyulingyue** [comment]
第一种，实际上使用的时候没有区别

**Reply by Gebbap** [reply]
多谢
